microsite/src/main/tut/overview/effects.md

```diff
@@ -5,3 +5,111 @@ title:  "Effects"
 ---
 
 # {{page.title}}
+`IO` describes the following effects:
+
+### Errors
+
+```scala
+val failedIO: IO[Exception, Nothing] =
```

Comment:
This is a good start! If you see other examples, you can see they use `tut` which will compile code and make sure there are no errors. Try to copy their example.

Reply:
When I'm using ```tut:silent for example, I'm getting an error:
<img width="581" alt="Screen Shot 2019-03-14 at 6 45 04 AM" src="https://user-images.githubusercontent.com/3358827/54332079-4234c680-4625-11e9-850d-b54742821e43.png">
<img width="690" alt="Screen Shot 2019-03-14 at 6 44 43 AM" src="https://user-images.githubusercontent.com/3358827/54332085-4a8d0180-4625-11e9-988c-9813473776e6.png">